apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -863,6 +863,12 @@ const styles = {
 // The "new student rows" should always be next.
 // This function takes into account having multiple "add rows"
 export const sortRows = (data, columnIndexList, orderList) => {
+  // Add secondary and tertiary sorting factors
+  columnIndexList.push('name');
+  orderList.push(orderList[0]);
```

Comment:
Does the original `orderList` input change depending on what column the end-user selects? Is it intentional that we want every column to have the same sorting value?

Reply:
Got it. So if for example its ascending, then we want `name` and `id` to be ascending as well?